### ISSUE TITLE
Typeerror

### ISSUE THREAD

**Issue by mcavusoglumert** [tmb5cg/Fifa-Autobidder#46]
C:\Users\MERT\Desktop\FUT\Fifa-Autobidder-main>python src/run.py
Traceback (most recent call last):
  File "src/run.py", line 16, in <module>
    app = GUI(root)
  File "C:\Users\MERT\Desktop\FUT\Fifa-Autobidder-main\src\gui.py", line 101, in __init__
    self.initialize_driver()
  File "C:\Users\MERT\Desktop\FUT\Fifa-Autobidder-main\src\gui.py", line 270, in initialize_driver
    self.driver = create_driver()
  File "C:\Users\MERT\Desktop\FUT\Fifa-Autobidder-main\src\helpers.py", line 108, in create_driver
    driver = uc.Chrome(options=options)
  File "C:\Users\MERT\AppData\Local\Programs\Python\Python38-32\lib\site-packages\undetected_chromedriver\__init__.py", line 409, in __init__
    browser = subprocess.Popen(
  File "C:\Users\MERT\AppData\Local\Programs\Python\Python38-32\lib\subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\MERT\AppData\Local\Programs\Python\Python38-32\lib\subprocess.py", line 1247, in _execute_child
    args = list2cmdline(args)
  File "C:\Users\MERT\AppData\Local\Programs\Python\Python38-32\lib\subprocess.py", line 549, in list2cmdline
    for arg in map(os.fsdecode, seq):
  File "C:\Users\MERT\AppData\Local\Programs\Python\Python38-32\lib\os.py", line 818, in fsdecode
    filename = fspath(filename)  # Does type-checking of `filename`.
TypeError: expected str, bytes or os.PathLike object, not NoneType

C:\Users\MERT\Desktop\FUT\Fifa-Autobidder-main>

I'm getting this error, any ideas on a fix?

**Comment by yannmazita** [comment]
This might come from the 32-bit version of Python you're using, please run this code snippet

```python
import os
m = map(os.environ.get, ("PROGRAMFILES", "PROGRAMFILES(X86)"))

for i in m:
    print(i)
```
and then post the result here.

**Comment by mcavusoglumert** [comment]
PS C:\Users\MERT\Desktop\FUT>  & 'C:\Users\MERT\AppData\Local\Programs\Python\Python38-32\python.exe' 'c:\Users\MERT\.vscode\extensions\ms-python.python-2022.16.1\pythonFiles\lib\python\debugpy\adapter/../..\debugpy\launcher' '63277' '--' 'c:\Users\MERT\Desktop\FUT\dasdasd.py'
C:\Program Files (x86)
C:\Program Files (x86)
PS C:\Users\MERT\Desktop\FUT>

**Comment by yannmazita** [comment]
Indeed this is caused by a bug in undetected-chromedriver (<3.2.0) on Python 32 bit, [see Issue 497](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/497#issuecomment-1039864761) and [PR 920](https://github.com/ultrafunkamsterdam/undetected-chromedriver/pull/920)

It should have been fixed with version 3.2.0. You can either update your package or switch to Python 64 bit if possible.
